app/controllers/grade_entry_forms_controller.rb

```diff
@@ -47,6 +47,22 @@ def view_summary
     @date = params[:date]
   end
 
+  # returns the column distribution/breakdown for each grade_entry_item in a grade_entry_form
+  #   (see the second chart in the summary of a mark's spreadsheet, accessed from the dashboard)
+  def column_breakdown
+    @grade_entry_form = GradeEntryForm.find(params[:id])
+    @grade_entry_items = @grade_entry_form.grade_entry_items unless @grade_entry_form.nil?
```

Comment:
The `nil` check isn't necessary; `.find` raises an error if the grade entry form isn't found.

Also see my comment about the `@`, and about how you should be formatting the JSON in the response.